Gemfile

```diff
@@ -4,11 +4,16 @@ gemspec
 
 group :test do
   gem 'rubocop', require: false
+  gem 'bundler', '~> 1.9'
+  gem 'pry'
```

Comment:
pry should have stayed in the development group. It should never be used in a test in CI